starter-code/routes/celebrities.js

```diff
@@ -0,0 +1,61 @@
+const express = require('express')
+
+const celebrityModel = require('../models/celebrity')
+
+const router = express.Router()
+
+
+router.get('/celebrities', (req,res, next)=>{
+    celebrityModel.find()
+    .then((celebrity) =>{
+        console.log(celebrity)
+        res.render('celebrities/index.hbs', {celebrity})
+    })
+    .catch((err) =>{
+        console.log("Problem with the collection", err)
+    })
+})
+
+
+router.get('/celebrities/new', (req,res, next)=>{
+    res.render('celebrities/new.hbs')
+
+ })
+
+router.post('/celebrities/new', (req, res) =>{
+ celebrityModel.create(req.body)
+ .then(()=>{
+     res.redirect('/celebrities')
+ })
+ .catch(()=>{
+     res.redirect('/celebrities/new');
+     console.log('Couldnt create celebrity')
```

Comment:
put your console.logs (and any other code) before any render, redirect or next. As otherwise they might not be executed.